src/complex/Common/Range2D.cpp

```diff
@@ -31,25 +31,25 @@ Range2D::RangeType Range2D::getRange() const
 // -----------------------------------------------------------------------------
 size_t Range2D::minRow() const
 {
-  return m_Range[0];
+  return m_Range[2];
```

Comment:
Not required, but could be useful to document what these index represent next to the member variable such as:
`auto m_Range = {}; // layout: { min Column, max Column, min Row, max Row } `
and then next to the return statements just mention the documentation like:
`return m_Range[0]; // see member declaration for details`

Reply:
The layout is documented in the header